runtime/src/main/java/org/corfudb/util/MetricsUtils.java

```diff
@@ -128,6 +166,8 @@ public static void metricsReportingSetup(@NonNull MetricRegistry metrics) {
 
         if (metricsCollectionEnabled) {
             setupCsvReporting(metrics);
+            setupJvmMetrics(metrics);
+            setupLogAnalysisMetrics(metrics);
```

Comment:
What is LogAnalysis and how is it different from reporting?

Reply:
It provides an aggregate to the error/warning/etc messages being logged. More detail in the below comment. If you're not convinced of the value, I will remove this.